script.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable linebreak-style */
```

Comment:
- [ ] I think you shouldn't disable linters rules, you should fix them instead

Reply:
Addressed this.